A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml

```diff
@@ -107,7 +107,7 @@
          <CheckpointDesc SubjectTT="TT:Exterior Lights" ExpectationTT="TT:As required" />
       </Checkpoint>
       <Checkpoint Id="A320_CPREP1_NO_SMOKING_SEATBELTS">
-         <CheckpointDesc SubjectTT="TT:No Smoking and Seatbelt Signs" ExpectationTT="TT:AUTO" />
+         <CheckpointDesc SubjectTT="TT:No Smoking and Seatbelt Signs" ExpectationTT="TT:ON" />

```

Comment:
So, its tricky, seat belts only have on and off, and should be turned on in this step, but no smoking does have an auto, and it should be auto in this step, not on (i believe)

Reply:
I will seperate the two

Reply:
I think you reversed the ON and AUTO eventually now @DoToAdventures, I swapped it in my latest pull (https://github.com/flybywiresim/a32nx/pull/563)